continuous_integration/travis/install.sh

```diff
@@ -42,7 +42,7 @@ if [[ ${UPSTREAM_DEV} ]]; then
         locket \
         git+https://github.com/pydata/sparse \
         git+https://github.com/dask/s3fs \
-        git+https://github.com/intake/fsspec \
+        git+https://github.com/intake/filesystem_spec \
```

Comment:
Is this actually getting built? I would have noticed, I think

Reply:
It's built nightly.

On PRs, you can trigger a build by including `test-upstream` in the message.

Reply:
good to know.